### ISSUE TITLE
ERR! FAILED: @storybook/vue@3.2.16 when installing forked storybook

### ISSUE THREAD

**Issue by folmert** [storybooks/storybook#2382]
Steps to reproduce:

1. Fork this repository.

2. `cd storybook/app/vue`

3. `npm i -g @storybook/cli`

```
npm WARN deprecated babel-preset-es2015@6.24.1: 🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read babeljs.io/env to update!
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
C:\Users\LFOLMERT\AppData\Roaming\npm\getstorybook -> C:\Users\LFOLMERT\AppData\Roaming\npm\node_modules\@storybook\cli\bin\generate.js
+ @storybook/cli@3.2.16
updated 13 packages in 14.84s
```

4. `getstorybook`

```
 • Detecting project type. ✓
 • Adding storybook support to your "Single File Components Vue" app. ✓
 • Preparing to install dependencies. ✓

yarn install v1.2.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@1.1.3: The platform "win32" is incompatible with this module.
info "fsevents@1.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@1.1.2: The platform "win32" is incompatible with this module.
info "fsevents@1.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
warning "@storybook/addon-storyshots@3.2.16" has unmet peer dependency "babel-core@^6.26.0".
warning "@storybook/react-native@3.2.16" has unmet peer dependency "react-native@>=0.27.0".
warning "@storybook/vue@3.2.16" has unmet peer dependency "vue@*".
warning "@storybook/vue@3.2.16" has unmet peer dependency "vue-loader@*".
warning "@storybook/vue@3.2.16" has unmet peer dependency "vue-template-compiler@*".
warning "graphiql@0.11.10" has unmet peer dependency "prop-types@>=15.5.0".
warning "react-datetime@2.11.0" has unmet peer dependency "moment@>=2.16.0".
warning "babel-loader@7.1.2" has unmet peer dependency "babel-core@6 || 7 || ^7.0.0-alpha || ^7.0.0-beta || ^7.0.0-rc".
warning "react-native@0.50.4" has incorrect peer dependency "react@16.0.0".
warning "babel-preset-react-app@3.1.0" has unmet peer dependency "babel-runtime@^6.23.0".
warning "dotenv-webpack@1.5.4" has unmet peer dependency "webpack@^1 || ^2 || ^3".
warning "vue-loader@13.5.0" has unmet peer dependency "css-loader@*".
warning "webpack-dev-server@2.9.4" has unmet peer dependency "webpack@^2.2.0 || ^3.0.0".
warning "codemirror-graphql@0.6.11" has incorrect peer dependency "graphql@^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.8.0 || ^0.9.0 || ^0.10.0".
warning "react-icon-base@2.1.0" has unmet peer dependency "prop-types@*".
warning "webpack-dev-middleware@1.12.0" has unmet peer dependency "webpack@^1.0.0 || ^2.0.0 || ^3.0.0".
[5/5] Building fresh packages...
$ node ../../scripts/prepare.js
ERR! FAILED: @storybook/vue@3.2.16
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

 • Installing dependencies. ✖

     An error occurred while installing dependencies.
```

5. `yarn storybook`
```
yarn run v1.2.1
$ start-storybook -p 6006
module.js:529
    throw err;
    ^
Error: Cannot find module '../dist/server'
    at Function.Module._resolveFilename (module.js:527:15)
    at Function.Module._load (module.js:476:23)
    at Module.require (module.js:568:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\LFOLMERT\projects\storybook-test\storybook\app\vue\bin\index.js:3:1)
    at Module._compile (module.js:624:30)
    at Object.Module._extensions..js (module.js:635:10)
    at Module.load (module.js:545:32)
    at tryModuleLoad (module.js:508:12)
    at Function.Module._load (module.js:500:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by folmert** [comment]
Output from debug.log:

```
35368 silly lifecycle @storybook/vue@3.2.16~prepare: Args: [ '/d /s /c', 'node ../../scripts/prepare.js' ]
35369 silly lifecycle @storybook/vue@3.2.16~prepare: Returned: code: 1  signal: null
35370 info lifecycle @storybook/vue@3.2.16~prepare: Failed to exec prepare script
35371 verbose stack Error: @storybook/vue@3.2.16 prepare: `node ../../scripts/prepare.js`
35371 verbose stack Exit status 1
35371 verbose stack     at EventEmitter.<anonymous> (C:\Program Files\nodejs_8\node_modules\npm\node_modules\npm-lifecycle\index.js:280:16)
35371 verbose stack     at emitTwo (events.js:125:13)
35371 verbose stack     at EventEmitter.emit (events.js:213:7)
35371 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs_8\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
35371 verbose stack     at emitTwo (events.js:125:13)
35371 verbose stack     at ChildProcess.emit (events.js:213:7)
35371 verbose stack     at maybeClose (internal/child_process.js:927:16)
35371 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
35372 verbose pkgid @storybook/vue@3.2.16
35373 verbose cwd C:\Users\LFOLMERT\projects\storybook\app\vue
35374 verbose Windows_NT 10.0.14393
35375 verbose argv "C:\\Program Files\\nodejs_8\\node.exe" "C:\\Program Files\\nodejs_8\\node_modules\\npm\\bin\\npm-cli.js" "install"
35376 verbose node v8.7.0
35377 verbose npm  v5.4.2
35378 error code ELIFECYCLE
35379 error errno 1
35380 error @storybook/vue@3.2.16 prepare: `node ../../scripts/prepare.js`
35380 error Exit status 1
35381 error Failed at the @storybook/vue@3.2.16 prepare script.
35381 error This is probably not a problem with npm. There is likely additional logging output above.
35382 verbose exit [ 1, true ]
```

**Comment by vpandura** [comment]
Getting the same error!! 

``npm WARN react-native@0.50.4 requires a peer of react@16.0.0 but none was installed
``

**Comment by Hypnosphi** [comment]
You need to run `yarn bootstrap --core` in the root to install all the dependencies, cross-link and transpile the local packages. See contributors guide: https://github.com/storybooks/storybook/blob/master/CONTRIBUTING.md

By the way, the examples for Vue Storybook live in `examples/vue-kitchen-sink`, so you shouldn't need to add storybook in `app/vue` directory

**Comment by folmert** [comment]
Actually the error is gone after I removed yarn, it works when installing with npm, so might it be a yarn issue?

**Comment by Hypnosphi** [comment]
This project uses yarn workspaces feature: https://yarnpkg.com/blog/2017/08/02/introducing-workspaces/

So when you switch to npm, you actually install the published `@storybook/*` packages instead of linking the local ones. Note that you still need to install root packages

**Comment by ChristianDavis** [comment]
There was a linking issue in yarn 1.0.0 that is corrected in 1.3.0 (released as 1.3.2), https://github.com/yarnpkg/yarn/releases/tag/v1.3.0

1.3.2 should be listed as the lowest compatible version in the guide https://github.com/storybooks/storybook/pull/2638#issue-285854674
